### PR TITLE
Adjust statuses and tooltips for functions and pods

### PR DESCRIPTION
**Pull request by dariadomagala**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

**Review by Wawrzyn321 — CHANGES_REQUESTED** [review]
(writing here in case me forgets)
We could remove full stops after error statuses, so we have `Error: message` instead of `Error.: message`.

**Review by Wawrzyn321 — APPROVED** [review]

File: core/src/i18n/en.yaml [hunk 1]
```diff
@@ -1370,33 +1376,53 @@ pods:
...
+    podinitializing: Initializing
+    poststarthookerror: Error
+    registryunavailable: Error
+    runcontainererror: Error
     running: Running
     terminated: Terminated
     terminating: Terminating
   title: Pods
   tooltips:
+    completed: Pod has completed successfully.
```
Comment (mmitoraj) [inline core/src/i18n/en.yaml:1402]: What do you mean? Has it restarted successfully?

File: core/src/i18n/en.yaml [hunk 2]
```diff
@@ -1370,33 +1376,53 @@ pods:
...
+    containersnotinitialized: Containers are not initialized.
+    containersnotready: Containers are not ready.
+    crashloopbackoff: Pod is in a crashing loop
+    createcontainerconfigerror: Create container configuration error
+    createcontainererror: Create container error
+    dockerdaemonnotready: Docker Daemon is not ready.
+    errimageneverpull: Image has never been pulled
+    errimagepull: Image pull error
     error: Pod failed to start
+    evicted: Pod has been evicted.
```
Comment (mmitoraj) [inline core/src/i18n/en.yaml:1413]:
```suggestion
    evicted: Pod has been removed.
```

File: core/src/i18n/en.yaml [hunk 3]
```diff
@@ -1370,33 +1376,53 @@ pods:
...
+    imageinspecterror: Image inspect error
+    imagepullbackoff: Could not pull a container image
+    invalidimagename: Invalid image name
+    networkpluginnotready: Network plugin is not ready
     no-restarts: None of the containers has restarted.
-    running: Pod is running correctly
+    podinitializing: Pod is being initialized.
```
Comment (mmitoraj) [inline core/src/i18n/en.yaml:1419]:
```suggestion
    podinitializing: Initializing the Pod.
```

File: core/src/i18n/en.yaml [hunk 4]
```diff
@@ -1370,33 +1376,53 @@ pods:
...
+    poststarthookerror: Post-start hook error
+    registryunavailable: Registry unavailable
+    runcontainererror: Run container error
+    running: Pod is running correctly.
+    terminated: Pod has terminated.
+    terminating: Pod is being terminated.
```
Comment (mmitoraj) [inline core/src/i18n/en.yaml:1425]:
```suggestion
    terminating: Terminating the Pod.
```

**Review by mmitoraj — COMMENTED** [review]
I added some suggestions. I have a question regarding the full stops a the end of some sentences. Maybe you could show me the tooltips in the UI so that we can confirm that it displays fine?

**Review by Wawrzyn321 — DISMISSED** [review]

**Review by Wawrzyn321 — APPROVED** [review]

**Review by mmitoraj — APPROVED** [review]

**Pull request merged**